extension-manifest-v3/src/content_scripts/cosmetics.js

```diff
@@ -10,4 +10,13 @@
  */
 import { injectCosmetics } from '@cliqz/adblocker-webextension-cosmetics';
 
-injectCosmetics(window);
+function getCosmeticsFilters(args) {
+  chrome.runtime.sendMessage({
+    action: 'getCosmeticsFilters',
+    ...args,
+  });
+
+  return {};
+}
+
+injectCosmetics(window, true, getCosmeticsFilters);
```

Comment:
I think, getCosmeticFilters needs to return a promise. While testing, I got this error on spiegel.de:

```
cosmetics.js:600 Uncaught TypeError: getCosmeticsFilters(...).then is not a function
    at injectCosmetics (cosmetics.js:600:86)
    at cosmetics.js:647:5
    at cosmetics.js:649:3
```

I think that is the code in the library that throws:
https://github.com/ghostery/adblocker/blob/692b84f6d2de57034e5a99e2a17fbcc3570126c6/packages/adblocker-webextension-cosmetics/adblocker.ts#L259